runners/spark/src/main/java/org/apache/beam/runners/spark/translation/TransformTranslator.java

```diff
@@ -173,19 +181,21 @@ public void evaluate(Combine.GroupedValues<K, VI, VO> transform, EvaluationConte
 
   private static final FieldGetter COMBINE_GLOBALLY_FG = new FieldGetter(Combine.Globally.class);
 
-  private static <I, A, O> TransformEvaluator<Combine.Globally<I, O>> combineGlobally() {
-    return new TransformEvaluator<Combine.Globally<I, O>>() {
+  private static <InputT, ArrayT, OutputT> TransformEvaluator<Combine.Globally<InputT, OutputT>>
+  combineGlobally() {
+    return new TransformEvaluator<Combine.Globally<InputT, OutputT>>() {
 
       @Override
-      public void evaluate(Combine.Globally<I, O> transform, EvaluationContext context) {
-        final Combine.CombineFn<I, A, O> globally = COMBINE_GLOBALLY_FG.get("fn", transform);
+      public void evaluate(Combine.Globally<InputT, OutputT> transform, EvaluationContext context) {
+        final Combine.CombineFn<InputT, ArrayT, OutputT> globally =
```

Comment:
AccumT instead of ArrayT is more accurate and inline with the SDK's `Combine.CombineFn`